e2e/global.d.ts

```diff
@@ -0,0 +1,3 @@
+// Set in puppeteer-custom-environment.ts. Read in jest.test-setup.ts
+declare const __TEST_NAMES__: string[];
+declare const __SPEC_NAME__: string;
```

Comment:
They're jest global variables.  Naming convention I used is double underscores before and after the name.